iron-form.html

```diff
@@ -207,16 +207,36 @@
        * @return {void}
        */
       attached: function() {
-        this._nodeObserver = Polymer.dom(this).observeNodes(
-          function(mutations) {
-            for (var i = 0; i < mutations.addedNodes.length; i++) {
-              if (mutations.addedNodes[i].tagName === 'FORM' && !this._alreadyCalledInit) {
-                this._alreadyCalledInit = true;
-                this._form = mutations.addedNodes[i];
-                this._init();
+        // We've already found the `<form>`, return.
```

Comment:
lol i had meant more along the lines of _why_ would we have found this already (if we've detached and reattached); otherwise this comment literally just...says `if ()..return`